src/field_theory/splitting_field.lean

```diff
@@ -480,23 +480,21 @@ theorem nat_degree_remove_factor' {f : K[X]} {n : ℕ} (hfn : f.nat_degree = n+1
 by rw [nat_degree_remove_factor, hfn, n.add_sub_cancel]
 
 /-- Auxiliary construction to a splitting field of a polynomial. Uses induction on the degree. -/
```

Comment:
```suggestion
/-- Auxiliary construction to a splitting field of a polynomial, which removes `n` factors. -/
```
or something similar to explain what `n` means.

Reply:
```suggestion
/-- Auxiliary construction to a splitting field of a polynomial, which removes `n` (arbitrarily-chosen) factors.

Uses recursion on the degree. For better definitional behaviour, structures including `splitting_field_aux`
(such as instances) should be defined using this recursion in each field, rather than defining the whole
tuple through recursion.
-/
```